packages/ui/Scrollable/Scrollable.js

```diff
@@ -823,22 +804,44 @@ class ScrollableBase extends Component {
 
 	// scroll start/stop
 
-	start ({targetX, targetY, animate = true, duration = animationDuration}) {
+	start ({targetX, targetY, animate = true, duration = animationDuration, overscrollEffect = true}) {
 		const
-			{scrollLeft, scrollTop} = this,
-			bounds = this.getScrollBounds();
+			{isDragging, scrollLeft, scrollTop} = this,
+			bounds = this.getScrollBounds(),
+			{maxLeft, maxTop} = bounds;
 
 		this.animator.stop();
 		if (!this.scrolling) {
 			this.scrolling = true;
 			this.forwardScrollEvent('onScrollStart');
 		}
 
-		if (Math.abs(bounds.maxLeft - targetX) < epsilon) {
-			targetX = bounds.maxLeft;
+		if (Math.abs(maxLeft - targetX) < epsilon) {
+			targetX = maxLeft;
+		}
+		if (Math.abs(maxTop - targetY) < epsilon) {
+			targetY = maxTop;
 		}
-		if (Math.abs(bounds.maxTop - targetY) < epsilon) {
-			targetY = bounds.maxTop;
+
+		if (overscrollEffect) {
+			if (scrollLeft !== targetX && this.canScrollHorizontally(bounds)) {
+				const edge = this.isOverEdge(targetX, maxLeft);
+
+				if (isDragging) {
+					this.applyOverscrollEffectOnDrag('horizontal', edge, targetX, overscrollTypeHold);
```

Comment:
Do we need to call `applyOverscrollEffectOnDrag` when `edge` is equals to `null`?